src/test/lifecycle/common.js

```diff
@@ -512,3 +539,13 @@ exports.test_and_prefix_size = async function(Bucket, Key, s3) {
     assert(actualFilter.ObjectSizeGreaterThan === expectedFilter.ObjectSizeGreaterThan, 'and prefix size filter - ObjectSizeGreaterThan');
     assert(actualFilter.ObjectSizeLessThan === expectedFilter.ObjectSizeLessThan, 'and prefix size filter - ObjectSizeLessThan');
 };
+
+exports.test_expiration_date_in_past = async function(Bucket, Key, s3) {
+    const putLifecycleParams = past_date_lifecycle_configuration(Bucket, Key);
+
+    await s3.putBucketLifecycleConfiguration(putLifecycleParams)
+        .catch(error => {
+            assert(error.code === 'InvalidArgument', 'Expected InvalidArgument: expiration date in the past');
+            console.log('Expected error received, expiration date in the past');
+        });
```

Comment:
Is there a special reason for using the promise chain here?

Reply:
Added try/catch instead of promise chain. Thanks